features/step_definitions/report_steps.rb

```diff
@@ -1,9 +1,18 @@
-Given /^test result report (.+) was generated(?: with UID ([a-zA-Z0-9_-]+))? for organization (.+)$/ do |name,uid,organization_name|
+Given /^test result report (.+) was generated(?: (\d+) day(?:s)? ago)?(?: with UID ([a-zA-Z0-9_-]+))? for organization (.+)$/ do |name,days,uid,organization_name|
   options = {
     uid: uid,
     organization: named_record(organization_name)
   }
 
+  if days
+    date = days.to_i.days.ago
+
+    options.merge!({
+      started_at: date - 3.minutes,
```

Comment:
Should I change also there ? https://github.com/probedock/probedock/blob/master/spec/factories/test_report.rb#L21-L22